src/strongbox/core.clj

```diff
@@ -775,18 +821,23 @@
                :matched :addon/toc+summary+match)]
   (logging/with-addon addon
     (let [expanded-addon (when (:matched? addon)
+                           (joblib/tick-delay 0.25) ;; todo: interleave form?
```

Comment:
create an interleave thingy and remove these manual ticks if possible

Reply:
I created an interleave thingy:

```clojure
(defmacro tick-form
  [& form]
  (let [num-forms (count form)
        p (/ 1.0 num-forms)]
    `(interleave form
                 ~@(for [~i (range 1 (inc num-forms))]
                     (joblib/tick ~(* i p))))))
```

unfortunately it can't handle bindings. 

switching to something like this might work: https://github.com/ztellman/riddley but would require replacing the simple `for` loop I have and ... eh. no energy for that.